script.js

```diff
@@ -77,3 +85,25 @@ add.addEventListener('click', () => {
     localStorage.setItem('books', JSON.stringify(booksBox1.getBooks));
   }
 });
+// Menu interactive
+const listMn = document.getElementById('list');
+const addMn = document.getElementById('add-new');
+const contactMn = document.getElementById('contact');
+const allBooks = document.querySelector('.all-books');
+const addBook = document.querySelector('.add-book');
+const contact = document.querySelector('.contact');
+listMn.addEventListener('click', () => {
+  allBooks.classList.remove('hidden');
+  addBook.classList.add('hidden');
+  contact.classList.add('hidden');
+});
+addMn.addEventListener('click', () => {
+  allBooks.classList.add('hidden');
+  addBook.classList.remove('hidden');
+  contact.classList.add('hidden');
+});
+contactMn.addEventListener('click', () => {
+  allBooks.classList.add('hidden');
+  addBook.classList.add('hidden');
+  contact.classList.remove('hidden');
+});
```

Comment:
- [ ] You've both done a great job I must say, but I noticed that the `hidden `class is not effective for the three sections. ℹ️ 

## Your Project
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/211786241-6738fd71-91eb-41f6-bd5f-4dc6a5a4aaa8.png)